modules/ossm-cert-manager-integration.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration_{context}"]
+== Integrating cert-manager with Service Mesh
+
+Cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
+
+It obtains certificates from a variety of Issuers, both popular public Issuers as well as private Issuers, and ensures the certificates are valid and up-to-date. It also attempts to renew certificates at a configured time before expiry.
+
+.Prerequisites
+
+//There are instances when you want to use cert-manager instead of general certificates or default certificates. OpenShift Service Mesh requires cert-manager to provide:
+
+* A TLS secret with an intermediate CA key and certificate for `istiod`:
+[source, yaml]
+----
+name: istiod-tls;
+namespace: the same one in which SMCP is deployed;
+data keys: tls.key and tls.crt.
+----
+
+* A config map with a root certificate:
+[source, yaml]
+----
+name: istio-ca-root-cert or custom specified in spec.security.certificateAuthority.cert-manager.rootCAConfigMapName;
```

Comment:
We already have plans to mark `rootCAConfigMapName` as depricated in [OSSM-3376](https://issues.redhat.com/browse/OSSM-3376). @pbajjuri20 what do you think? It's not technically depricated yet, but if we are just now introducing something we are already planning to depricate I feel like we should reconsider putting it in the docs. At the very least I think there should be a warning about using it.

Reply:
I think if is not deprecated should still be in the document, and then in 2.4 documentation the mention that is going to be deprecated. That's what I think

Reply:
If `rootCAConfigMapName` is going to be deprecated when 2.4 is released, then does this part need to come out, or is it being replaced with something else?

Reply:
Maybe the best way is to set a comment saying "to be deprecated in 2.4"